core/internal/pathtree/pathtree.go

```diff
@@ -1,5 +1,7 @@
 package pathtree
 
+import "github.com/wandb/segmentio-encoding/json"
```

Comment:
i assume you will remove it in your follow up PR right?

Reply:
yes---all these PRs are work I ended up having to do to remove it while minimizing unexpected breakages